backend/docs/installation/docker.md

```diff
@@ -1,20 +0,0 @@
-# Docker
-
```

Comment:
Pls remove the one at root instead.

Reply:
I ran `git rm -r docs/` command to delete that directory. How to revert back? `git reset "last-commit-hash"` not working.

Reply:
@abishekvashok I think docs should be present at root.

Reply:
I agree. Because root `docs` are linked in the Readme.md, deleting them would require edit in readme too.

Reply:
@vaibhavsingh97 the docs are exclusive to the backend at root, since we are going to treat the front-end and backend folders as independent repository they must have their own docs in the respective folder. 

The backend docs which are here that were deleted works for the backend, should we remove them?